modules/core/src/smithy4s/schema/SchematicRepr.scala

```diff
@@ -45,9 +45,8 @@ object SchematicRepr extends Schematic[Repr] {
 
   def document: String = "document"
 
-  def list[S](fs: String): String = s"list[$fs]"
-
-  def set[S](fs: String): String = s"set[$fs]"
+  def collection[C[_], S](tag: CollectionTag[C, S], fs: String): String =
+    s"collection[$fs]"
```

Comment:
call upon the [name](https://github.com/disneystreaming/smithy4s/pull/290/files#r913496776) method